src/ui/store/nodeSamplerPreviewStore.ts

```
@@ -0,0 +1,219 @@
+        if (!normalizedNodeId) continue;
+
+        workflowNodeIds.add(normalizedNodeId);
+        if (isKnownSamplerNodeDescriptor(node)) {
+          samplerNodeIds.add(normalizedNodeId);
+        }
+      }
+
+      const nextPreviews = new Map(state.previewsByNode);
+      for (const [nodeId, frames] of nextPreviews.entries()) {
+        if (workflowNodeIds.has(nodeId)) continue;
+        frames.forEach((frame) => revokeIfBlobUrl(frame.imageUrl));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)